plugin/RootIgnore.vim

```diff
@@ -22,7 +22,7 @@ function! s:WildignoreFromGitignore(gitpath, isAtRoot)
     let igstring = ''
     for oline in readfile(gitignore)
 
-      let line = substitute(oline, '\s|\n|\r', '', "g")
```

Comment:
the pipes need to be escaped to become regex 'or' operator.

why is the `\n` and `\r` therre in the first place if the readfile reads line by line